_gsocproposals/2022/proposal_CernVM-FS-firecracker-based-integration-tests.md

```diff
@@ -0,0 +1,57 @@
+---
+title: CernVM-FS integration test framework powered by Firecracker VM
+layout: gsoc_proposal
+project: CernVM
```

Comment:
Project difficulty, duration and mentor availability required:
difficulty: [medium or high]
duration: [350 or 175 hours]
mentor_avail: [month interval]